lib/test_boosters/boosters/rspec.rb

```diff
@@ -28,7 +28,7 @@ def rspec_options
         @rspec_options ||= begin
           output_formatter = ENV.fetch("TB_RSPEC_FORMATTER", "documentation")
           # rubocop:disable LineLength
-          "#{ENV["TB_RSPEC_OPTIONS"]} --format #{output_formatter} --require #{formatter_path} --format SemaphoreFormatter --out #{report_path}"
+          "#{ENV["TB_RSPEC_OPTIONS"]} --format #{output_formatter} --require #{formatter_path} --format SemaphoreFormatter --out #{report_path} --profile"
```

Comment:
Couldn't `--profiler` be passed as part of `TB_RSPEC_OTPIONS` and in that way not be hardcoded to on?